docker/start_script.sh

```diff
@@ -0,0 +1,7 @@
+#!bin/bash
+
+# change python environment
+source activate emission
+
+# launch the webapp
+./e-mission-py.bash emission/net/api/cfc_webapp.py
```

Comment:
And launch it?

Reply:
This only occurs when we do `docker run`